cmd/tptctl/cmd/config_aws_cloud_account.go

```diff
@@ -113,9 +119,9 @@ var ConfigAwsCloudAccountCmd = &cobra.Command{
 			),
 			roleName,
 			awsAccountId,
-			provider.GetResourceManagerRoleName(requestedControlPlane),
+			externalRoleName,
 			*createdAwsAccount.ExternalId,
-			false,
+			true,
```

Comment:
This boolean enables the attachment of the `assume-any-role` policy, which allows anyone in an account to assume the created role by default.